test/higgstautau_cfg_80X_Aug16.py

```diff
@@ -1359,10 +1359,10 @@
 
 #from ICAnalysis.MVAMETPairProducer.mvaPFMET_cff_leptons_74X import mvaMetPairs
 
-#if release in ['80XMINIAOD']:
-#  process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
+if release in ['80XMINIAOD']:
+  process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
```

Comment:
sorry i though i already reverted this. It should be the master version now. I don't need my copy